worldedit-core/src/main/java/com/boydti/fawe/util/MainUtil.java

```diff
@@ -34,6 +34,9 @@
 import net.jpountz.lz4.LZ4Utils;
 import org.jetbrains.annotations.NotNull;
 
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
```

Comment:
Replace all instances of `javax.annotation.Nonnull` and `javax.annotation.Nullable` to JetBrains annotations.